internal/services/mssql/mssql_database_resource_test.go

```diff
@@ -1051,7 +1059,19 @@ func (MsSqlDatabaseResource) Exists(ctx context.Context, client *clients.Client,
 	return pointer.To(resp.Model != nil), nil
 }
 
+// Sets the tests 'Primary' and 'Secondary' locations based on the 'ARM_MICROSOFT_TEST' environment variable due to subscription quota policies
+func setTestDataLocationBySubscription(data acceptance.TestData) acceptance.TestData {
```

Comment:
I don't think we want this living in the provider as it seems subscription specific and we have ways of forcing locations through [this file](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/.teamcity/components/settings.kt#L135)

Reply:
Fixed.